ddtrace/contrib/django/patch.py

```diff
@@ -139,7 +145,17 @@ def traced_cache(django, pin, func, instance, args, kwargs):
             keys = utils.quantize_key_values(args[0])
             span.set_tag_str("django.cache.key", keys)
 
-        return func(*args, **kwargs)
+        result = func(*args, **kwargs)
```

Comment:
we should move this variable to where it's used
```suggestion
        result = func(*args, **kwargs)
        command_name = func.__name__
```

Reply:
fixed